Content.Server/GameObjects/Components/ActionBlocking/CuffableComponent.cs

```diff
@@ -115,7 +115,7 @@ public void AddNewCuffs(IEntity handcuff)
             _container.Insert(handcuff);
             CanStillInteract = _hands.Hands.Count() > CuffedHandCount;
 
-            OnCuffedStateChanged.Invoke();
+            OnCuffedStateChanged?.Invoke();
```

Comment:
This is the one bug that was revealed by this refactor. `AddNewCuffs` would work on `HumanMob_Content` because `StrippableComponent` would initialize `OnCuffedStateChanged`, but failed in isolation.